src/public/translations/en-SG/main.json

```diff
@@ -18,6 +18,7 @@
     "GO_GOV": "https://go.gov.sg",
     "POSTMAN_GOV": "https://postman.gov.sg",
     "SUPPORT_FORM_LINK": "https://go.gov.sg/formsg-support",
+    "TWILIO_SETUP_LINK": "https://go.gov.sg/form-twilio-setup",
```

Comment:
Should we name this "VERIFIED_SMS_SETUP_LINK` instead to decouple the naming from the service?

Reply:
yeap, good point - will change